src/ReactWrapper.js

```diff
@@ -468,18 +468,18 @@ export default class ReactWrapper {
    * testing events should be met with some skepticism.
    *
    * @param {String} event
-   * @param {Array} args
+   * @param {Object} mock (optional)
    * @returns {ReactWrapper}
    */
-  simulate(event, ...args) {
+  simulate(event, mock = {}) {
     this.single(n => {
       const mappedEvent = mapNativeEventNames(event);
       const eventFn = Simulate[mappedEvent];
       if (!eventFn) {
         throw new TypeError(`ReactWrapper::simulate() event '${event}' does not exist`);
       }
 
-      eventFn(findDOMNode(n), ...args);
+      eventFn(findDOMNode(n), mock);
```

Comment:
If that's how React TestUtils does it, it's probably fine. It'd definitely be an edge case where a user does need those second/third arguments. So we can wait until somebody brings up an issue needing it.